demisto_sdk/commands/common/constants.py

```diff
@@ -1219,6 +1239,30 @@ class PB_Status:
                                      'detectedhosts', 'modified', 'expiration', 'timestamp', 'shortdesc',
                                      'short_description', 'description', 'Tags', 'blocked']
 
+# TODO Should this list contain all specific fields?
```

Comment:
what is this list used for?

Reply:
It's used to check that a file is correct.